packages/playback-core/src/index.ts

```diff
@@ -30,6 +30,16 @@ export const generatePlayerInitTime = () => {
   return mux.utils.now();
 };
 
+/** @TODO REPLACE WITH mux.publicUtils.generateUUID() (CJP) */
+export const generateUUID = function () {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
```

Comment:
should we use `crypto.randomUUID()` when available?

Reply:
nah we should use `mux-embed`'s `generateUUID` for consistency imo (per callout). That said, we may want `mux-embed` to use `crypto.randomUUID()` under the hood when available if it isn't already.

Reply:
Missed that TODO, but yeah, mux-embed should use that API if available.